MinMaxTypeValues/MinMaxTypeValues/Program.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+
+namespace MinMaxTypeValues
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string byteType = "BYTE";
+            byte byteMinValue = byte.MinValue;
+            byte byteMaxValue = byte.MaxValue;
+
+            string shortType = "SHORT";
+            short shortMinValue = short.MinValue;
+            short shortMaxValue = short.MaxValue;
+
+            string intType = "INT";
+            int intMinValue = int.MinValue;
+            int intMaxValue = int.MaxValue;
+
+            string longType = "LONG";
+            long longMinValue = long.MinValue;
+            long longMaxValue = long.MaxValue;
+
+            string floatType = "FLOAT";
+            float floatMinValue = float.MinValue;
+            float floatMaxValue = float.MaxValue;
+
+            string doubleType = "DOUBLE";
+            double doubleMinValue = double.MinValue;
+            double doubleMaxValue = double.MaxValue;
+
+            string uintType = "UINT";
+            uint uintMinValue = uint.MinValue;
+            uint uintMaxValue = uint.MaxValue;
+
+            string ulongType = "ULONG";
+            ulong ulongMinValue = ulong.MinValue;
+            ulong ulongMaxValue = ulong.MaxValue;
+
+            Console.WriteLine
+                (
+                    "Minimal {0} data type value = {1},\n" +
```

Comment:
Concating strings it's very expensive operation for system. On small data, we do not notice drawdowns of performace. But if we will work with big requests or with database in a future it will be so long and it will be a trouble. So. for best performance you can use StringBuilder or string interpolation like:

1. 
 `Console.Writeline("hello {variable1}, my name is {variable2}", variable1, variable2);`
 
2. 
```
string variable1 ="....";
string variable2 = "....";
Console.Writeline($"hello {variable1, my name is {variable2}}")
```


    
More info about it you can find [here](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated)

Reply:
Please review changes: 'Remove extra variables (min/max datatype). Refactoring of Console.Writeline (remove concatination)'